test/functional/job/job_spec.lua

```diff
@@ -228,10 +228,10 @@ describe('jobs', function()
     it('returns a list of status codes', function()
       source([[
       call rpcnotify(g:channel, 'wait', jobwait([
-      \  jobstart([&sh, '-c', 'sleep 0.10; exit 4']),
-      \  jobstart([&sh, '-c', 'sleep 0.110; exit 5']),
-      \  jobstart([&sh, '-c', 'sleep 0.210; exit 6']),
-      \  jobstart([&sh, '-c', 'sleep 0.310; exit 7'])
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.10; exit 4']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.110; exit 5']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.210; exit 6']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.310; exit 7'])
```

Comment:
sleep isn't typically available on Windows 7. The old [`ping` trick](http://ss64.com/nt/sleep.html)

    ping -n 4 -w 110 127.0.0.1>nul

even with the `-w` option of ping it seems not very precise. We can ship `sleep.exe` with the Windows build, but even if we do that we should skip these tests if `sleep` cannot be found, so that running the tests on Windows is not unpleasant for new contributors. 

```c
if eval("executable('sleep')") == 0 then
  pending('missing "sleep" command', function() end)
else
  it('returns a list of status codes', function()
  ...
```

reference: https://github.com/neovim/neovim/wiki/Unit-tests